src/entity_extraction/training/spacy_ner/run_spacy_training.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env sh 
+
+# if running with conda envs, comment out if not
+conda activate fossil_lit
+
+echo python.__version__ = $(python -c 'import sys; print(sys.version)')
+# ensure we're in the MetaExtractor root directory
+echo "Current working directory: $(pwd)"
+
+# set the location of the labelled data, ideally this is run from root of repo
+DATA_DIR="sample_folder"
+TRAIN_SPLIT=0.7
+VAL_SPLIT=0.15
+TEST_SPLIT=0.15
+MODEL_PATH=""
+VERSION="v1"
+
+rm -f spacy_transformer_$VERSION.cfg
+
+python3 spacy_preprocess.py \
```

Comment:
See comments on this file below, should remove duplicate code, use `labelling_data_split.py` here then call pre-processing